pkg/api/render.go

```diff
@@ -88,6 +88,5 @@ func (hs *HTTPServer) RenderToPng(c *contextmodel.ReqContext) {
 		return
 	}
 
-	c.Resp.Header().Set("Content-Type", "image/png")
```

Comment:
the header is sent based on the file name anyway, so this will send png or pdf depending on the file name